src/backend/InvenTree/plugin/base/ui/api.py

```diff
@@ -54,6 +54,44 @@ def get(self, request):
         )
 
 
+class PluginDashboardList(APIView):
```

Comment:
Adding new api endpoints and implementing the same thing over and over again with different interfaces was what I was trying to avoid when I built the ui features implementation using a generic system in a very typesafe way. But that's up to you. Or @matmair what do you mean?

Reply:
Ah, ok I see. So a single endpoint for fetching *all* types of UI integrations? I can work on that.

Reply:
@wolflu05 please check this out again now - there is only a single API endpoint to access these features now